cms/templates/logo.pt

```diff
@@ -0,0 +1,9 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal"
+     metal:define-macro="logo">
+    <div id="logo-container" tal:define="localisation view.get_localisation()" tal:condition="localisation">
+        <a href="/" tal:condition="localisation.logo_image">
+            <img src="${view.get_image_url(localisation.logo_image_host, localisation.logo_image)}" border="0" alt="${localisation.logo_description}"/>
+        </a>
+        <div tal:condition="localisation.logo_text">${localisation.logo_text}</div>
```

Comment:
no, `logo_text` is a special case for MAMA at the moment. I'd leave it out from here and implement it on `unicore-cms-mama` override only.